docs/semgrep-app/findings.md

```diff
@@ -124,31 +149,43 @@ To **open findings**:
 4. Click **Open**.
 5. Optional: Include a **comment** explaining the action.
 
-To **close findings**, do one of the two options:
```

Comment:
Blocking: 

Reffering to these lines:
<img width="1129" alt="Screenshot 2022-06-20 at 13 41 50" src="https://user-images.githubusercontent.com/26669307/174594241-3917fca1-82a3-446a-8535-9deca3a25e4d.png">

Change both occurences of `one-by-one` to `one by one`.

Reference: https://dictionary.cambridge.org/dictionary/english/one-by-one

Reply:
(Added!)